Packager.hx

```diff
@@ -245,16 +230,23 @@ class Packager {
         }
 
         // Create setup.py file
-        var file = sys.io.File.write('_packages/connect.py/setup.py');
+        final file = sys.io.File.write('$outDir/setup.py');
+
+        file.writeString("from os import path" + EOL);
         file.writeString("from setuptools import setup" + EOL + EOL + EOL);
+        file.writeString("with open(path.join(path.abspath(path.dirname(__file__)), 'README.md')) as fhandle:" + EOL);
+        file.writeString("    README = fhandle.read()" + EOL + EOL + EOL);
         file.writeString("setup(" + EOL);
-        file.writeString("    name='connect'," + EOL);
+        file.writeString("    name='connect-sdk-haxe-port'," + EOL);
```

Comment:
This is the package name suggested by Max & Marc.

Reply:
Correct